src/Symfony/Bundle/FrameworkBundle/Logger/Logger.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Logger;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LogLevel;
+
+/**
+ * Minimalist PSR-3 logger designed to write in stdout, stderr or any other stream.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+final class Logger extends AbstractLogger
+{
+    private static $defaultOutputByLevel = array(
+        LogLevel::EMERGENCY => 'php://stderr',
+        LogLevel::ALERT => 'php://stderr',
+        LogLevel::CRITICAL => 'php://stderr',
+        LogLevel::ERROR => 'php://stderr',
+        LogLevel::WARNING => 'php://stderr',
+        LogLevel::NOTICE => false,
+        LogLevel::INFO => false,
+        LogLevel::DEBUG => false,
+    );
+    private $outputByLevel;
+    private $format;
+    private $dateFormat;
+    private $handles = array();
+
+    public function __construct(array $outputByLevel = array(), $format = '[%s] %s', $dateFormat = \DateTime::RFC3339)
```

Comment:
Given that this is a minimalist logger, could we make some decisions about the formats instead of making them configurable? In other words, hardcode `[%s] %s` as `$format` and `\DateTime::RFC3339` as `$dateFormat`. If someone wants configurability, use a full-featured logger such as Monolog.

Reply:
Why not, I've done that because it comes almost for free but we can be more restrictive. Exposing `$format` looks useful anyway.

Reply:
I think I agree with @javiereguiluz. Even if it's free, it should be crystal clear that this logger is just a default/simple one. For any configurability, people still need to install Monolog. So, I would remove any way that allow configuration of this class.